mantle/platform/qemu.go

```diff
@@ -471,8 +471,17 @@ type QemuBuilder struct {
 
 // NewQemuBuilder creates a new build for QEMU with default settings.
 func NewQemuBuilder() *QemuBuilder {
+	var defaultFirmware string
+	switch coreosarch.CurrentRpmArch() {
+	case "x86_64":
+		defaultFirmware = "bios"
+	case "aarch64":
+		defaultFirmware = "uefi"
+	default:
+		defaultFirmware = ""
```

Comment:
do we know what the string is for `ppc64le` and `s390x`?

Reply:
There isn't one today.  It's our own code that translates e.g. `uefi` into corresponding qemu options.